Segfault when running cnspec bundle lint

```
cnspec 8.8.0 (a1b05d1, 2023-05-02T11:41:39Z)
```

```
policies:
    - uid: test-best-practices
      authors:
        - email: test@test.co.za
          name: test Systems
      license: MPL-2.0
      name: test Best Practices
      query_counts: {}
      scoring_system: 2
      version: 1.0.2
      groups:
        - title: test Best Practices
          checks:
            - uid: test-ssh-service
              title: Get SSH services
              mql: |
                services.
                  where(name == /ssh/)
            - uid: test-ssh-config
              title: All the SSH config
              mql: |
                sshd.config.params
            - uid: ssh-authorized-keys
              title: Authorized Keys
              props:
                - uid: allowedKeys
                  title: Define allowed SSH keys
                  mql: |
                    return [
                    "AAAAB3NzaC1yc2EAAAADAQABAAABAQCmHoEOGM+gf1o1hoWiTP++Tg6Mt14K4IVsQ3cWEsUvOvZR5KfHejURa7899tpMB1s4BL11aH5"
                    ]
              mql: |
                authorizedkeys(path: '/home/user/.ssh/authorized_keys').map(_.key).containsOnly(props.allowedKeys)
```

`cnspec bundle lint policies.mql.yml`

Gives this error
```
→ lint policy bundle file=policies.mql.yml
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0xf0 pc=0x10a8c98e4]

goroutine 1 [running]:
go.mondoo.com/cnspec/internal/bundle.hasVariants(...)
	/home/runner/_work/cnspec/cnspec/internal/bundle/lint.go:464
go.mondoo.com/cnspec/internal/bundle.lintFile({0x1400058daa0, 0x27})
	/home/runner/_work/cnspec/cnspec/internal/bundle/lint.go:344 +0x1944
go.mondoo.com/cnspec/internal/bundle.Lint({0x14000a4c790?, 0x1, 0x1})
	/home/runner/_work/cnspec/cnspec/internal/bundle/lint.go:161 +0x200
go.mondoo.com/cnspec/apps/cnspec/cmd.glob..func3(0x111e91720?, {0x14000a47440, 0x1, 0x1?})
	/home/runner/_work/cnspec/cnspec/apps/cnspec/cmd/bundle.go:93 +0xfc
github.com/spf13/cobra.(*Command).execute(0x111e91720, {0x14000a473e0, 0x1, 0x1})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:920 +0x5b0
github.com/spf13/cobra.(*Command).ExecuteC(0x111e908c0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x35c
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
go.mondoo.com/cnspec/apps/cnspec/cmd.Execute()
	/home/runner/_work/cnspec/cnspec/apps/cnspec/cmd/root.go:80 +0x28
main.main()
	/home/runner/_work/cnspec/cnspec/apps/cnspec/cnspec.go:6 +0x1c
`****`
```

--- a/internal/bundle/lint.go
+++ b/internal/bundle/lint.go
@@ -460,8 +460,8 @@ func hasVariants(group *PolicyGroup, queryMap map[string]*Mquery) bool {
 		}
 
 		// check referenced query
-		q := queryMap[check.Uid]
-		if q.Variants != nil {
+		q, ok := queryMap[check.Uid]
+		if ok && q.Variants != nil {
 			return true
 		}
 	}

